File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/ui/view/CustTextView.java
package de.dimedis.mobileentry.ui.view;

import android.content.Context;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatTextView;

public class CustTextView extends AppCompatTextView {
    public CustTextView(Context context) {
        super(context);
    }

    public CustTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public CustTextView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public void setText(CharSequence text, BufferType type) {
        super.setText(text, type);
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/ui/view/CustTextView.kt
package de.dimedis.mobileentry.ui.view

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

class CustTextView : AppCompatTextView {
    constructor(context: Context?) : super(context!!)

    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs)

    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context!!, attrs, defStyleAttr)

    override fun setText(text: CharSequence, type: BufferType) {
        super.setText(text, type)
    }
}